src/chat/chat.js

```diff
@@ -1,115 +1,26 @@
 import Config from "../config/config.js"
+import Message, { NotifMessage, EventMessage } from "../config/message.js";
 
 export default class Chat {
-  static Events = {
-      1 : {
-          name: "random message",
-          message: "Hang in there buddy!"
-      },
-      2 : {
-          name: "water",
-          message: "Time for a water break!",
-          userInputs: {
-              opt1: "Yes! I am rehydrated"
-          }
-      },
-      3 : {
-          name: "stretch",
-          message: "How about a 5-min stretch?",
-          userInputs: {
-              opt1: "Done! Feeling much better now"
-          }
-      },
-      4 : {
-          name: "break",
-          message: "Wanna see something funny?",
-          userInputs: {
-              opt1: "A random meme",
-              opt2: "A random joke"
-          }
-      }
-  }
-
-  static BuddyProfilePicAttributes = {
-      src: "./assets/images/buddy-profile-pic-cat.png",
-      class: "buddy-chat-profile-pic",
-      alt: "buddy-profile-pic"
-  }
+  static ChatBoxDom = document.querySelector(".chat-content");
```

Comment:
```suggestion
  static DOM = document.querySelector(".chat-content");
```
Because to access this var, you would already indicate what it is: `Chat.DOM`